contracts/CourtPresaleActivate.sol

```diff
@@ -187,6 +185,15 @@ contract CourtPresaleActivate is IsContract, ApproveAndCallFallBack {
         _buyAndRegisterAsJuror(msg.sender, contributionTokenAmount, contributionToken, _activate);
     }
 
+    function _dataToBool(bytes memory _data) internal pure returns(bool) {
+        bool activate;
```

Comment:
Wondering if we can simplify this entirely to:

```solidity
function _hasData(bytes memory _data) internal pure returns (bool) {
  return _data.length > 0
}
```

Reply:
Definitely!!! :man_facepalming: Copy-pasting betrayed me here, haha. Way more elegant and simple this way.